README.md

```diff
@@ -141,16 +141,14 @@ Category Colors:
 #b7bdf8: pack
 -->
 
-This repository contains plugin configuration specifications, a collection of community-contributed plugins for AstroNvim, a neovim configuration by AstroNvim. The configuration specs are used to manage the various plugins that are used in AstroNvim.
+AstroNvim's community repository contains plugin configuration specifications — a collection of plugins contributed by the community for AstroNvim, a NeoVim configuration. These specifications help manage the variety of plugins used in AstroNvim.
 
 ## 📦 Setup
 
-Add the following to your `plugins` configuration, either in your `user/init.lua` file or in a file in `user/plugins/` folder such as `user/plugins/community.lua`
-If you're not sure what folders we are talking about it is recommended to read [AstroNvim Readme](https://docs.astronvim.com/Configuration/manage_user_config).
-
+To integrate the community plugins, add the provided configurations to your `plugins` setup. This can be done in your `user/init.lua` file or within a dedicated file in the `user/plugins/` directory. If you're unfamiliar with these directories, we recommend referring to the [AstroNvim Readme](https://docs.astronvim.com/Configuration/manage_user_config).
```

Comment:
```suggestion
To integrate the community plugins, add the provided configurations to your `plugins` setup. This can be done in your `user/init.lua` file or within a dedicated file in the `user/plugins/` directory. If you're unfamiliar with these directories, we recommend referring to the [AstroNvim documentation](https://docs.astronvim.com/Configuration/manage_user_config).
```